synapseAnnotations/data/tool.json

```diff
@@ -183,14 +183,14 @@
         "source": ""
       },
       {
-        "value": "containerRegistry",
-        "description": "",
-        "source": ""
+        "value": "containerImageRepository",
+        "description": "A location that hosts built container images",
+        "source": "Sage Bionetworks"
       }, 
       {
-        "value": "containerToolRegistry",
-        "description": "",
-        "source": ""
+        "value": "containerRegistry",
```

Comment:
https://developers.redhat.com/blog/2018/02/22/container-terminology-practical-introduction/ for inspiration?